reactDashboard/src/components/HeatMap/Heatmap.jsx

```diff
@@ -183,10 +186,11 @@ function Heatmap({
   ) {
     const counts = VALUETYPE_DEATHS === valueType ? deaths : cases;
     const total = VALUETYPE_DEATHS === valueType ? totalDeaths : totalCases;
+    const scotlandTotal = totalCount();
     return (
       <tr className="area" key={index}>
-        <td>{name}</td>
-        <td>{featureCode === FEATURE_CODE_SCOTLAND ? "" : total}</td>
+        <td className={featureCode === FEATURE_CODE_SCOTLAND ? "scotland-total": ""}>{name}</td>
```

Comment:
Might be cleaner to put the scotland-total class name on the tr element rather than repeating it

Reply:
Refactored